news/_posts/2020-05-11-cms-security-summit.md

```diff
@@ -0,0 +1,95 @@
+Everyone know how important and necessary it is for the community.
+
+We spent a lot of time during the summit talking about the self upgrade process, sure we will took time this year improving this module!
+
+## Disclose + Distribute + Query security bulletins in a standardized way?
+
+We paid the price at the beginning of the year, during weeks we sent and re sent emails to [cve.mitre.org](https://cve.mitre.org/) hoping they will update the [CVE-2017-9841](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-9841) about the PHPUnit issue in order to add the CGI/FastCGI context. We are still waiting...
+
+PSR-9 and PSR-10 are always on the side, this is the reason why the open source project decided to use and follow [GitHub Advisories](https://help.github.com/en/github/managing-security-vulnerabilities/about-github-security-advisories), and after weeks of usage, people look happy to see we are more transparent about security issues.
+
+But keep in mind we will never disclose how to reproduce security issues, you will never find any scripts in advisories or in CVE declared by PrestaShop, for your own goods.
+
+Of course it's not perfect, and we will never be. But we learn from the past, and we will do our best to listen the community and improve how we work.
+
+## Release window coordination
+
+To be compatible with PHP 7.3, we had no other choice than updating our dependencies, adding [BC Breaks](bc-breaks), etc. And we will certainly do the same for PHP 7.4 and so on.
+Moreover, PHP planned to release versions more often, which mean, in the future, many operating systems will have no choice than follow them, and PrestaShop too.
+We hope having the PHP community, Composer, and others famous people to the next summit.
+
+## Lighthouse score for security
+
+[LightHouse](https://developers.google.com/web/tools/lighthouse) is an automated tool for improving the quality of web pages. But something is missing around the security.
+The first thing can be an integration of [security.txt](https://securitytxt.org/) or [securityheaders.io](https://securityheaders.io) and allow people to submit report directly inside the browser.
+But why not adding a JavaScript scanner inside it too, and show outdated one.
+Whatever, Google prepare good things for this year 2020 :)
+
+
+## Other sessions
+
+Unfortunately, we don't have time to spend on all breakout sessions but we also had discussions around security tools, API, funding securituy improvements, static analysis and two factor authentication during breakout sessions. 
+
+# Thanks 
+
+We would like to say thanks to Google for organizing the event and to all participants for being active and passionate about improving security.
+Spending time with security folks has been very interesting, people know they need each other to go further.
+
+
+[bc-breaks]: https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/#bc-breaks
```

Comment:
Warning: this does not work on Build. https://github.com/PrestaShop/prestashop.github.io/issues/670